### PR TITLE
Printing unhandled messages into the server console

### PR DESCRIPTION
**Pull request by temMartik**

### CODE REVIEW

**Review by Pospelove — CHANGES_REQUESTED** [review]
1) Please fix the formatting
2) print **unknown** messages. you currently print **known** messages. it's good, but out of the original task's scope.
3) use debug instead of info
4) remove skymp5-client changes

**Review by Pospelove — CHANGES_REQUESTED** [review]
```
spdlog::debug("Got uncknown message from {}", rawMsgData.userId);
```
Please also write **JSON contents** of unknown messages.
Please remove all other changes since they are out of the task's scope. They **will** be hard to review and fix.

File: skymp5-client/src/remoteServer.ts [hunk 1]
```diff
@@ -382,8 +383,9 @@ export class RemoteServer implements MsgHandler, ModelSource, SendTarget {
       // TODO: move to a separate module
       once("update", () => Game.quitToMainMenu());
     }
-
+
```
Comment (nic11) [inline skymp5-client/src/remoteServer.ts:386]: Meaningless change

File: skymp5-server/cpp/server_guest_lib/PacketParser.cpp [hunk 2]
```diff
@@ -262,6 +270,8 @@ void PacketParser::TransformPacketIntoAction(Networking::UserId userId,
       break;
     }
     default:
-      throw PublicError("Unknown MsgType: " + std::to_string((TypeInt)type));
+      simdjson::dom::element data_;
+      ReadEx(jMessage, JsonPointers::data, &data_);
+      actionListener.OnUnknown(rawMsgData, data_);
```
Comment (nic11) [inline skymp5-server/cpp/server_guest_lib/PacketParser.cpp:275]: I think the whole object should be printed, not just data. (Probably with some generic validation and truncating so we won't spam console with huge messages with like 1K bytes or more)

File: skymp5-server/cpp/server_guest_lib/PacketParser.cpp [hunk 3]
```diff
@@ -155,18 +155,26 @@ void PacketParser::TransformPacketIntoAction(Networking::UserId userId,
...
-    case MsgType::UpdateProperty:
-      break;
-    case MsgType::PutItem:
+    case MsgType::UpdateProperty: {
+      uint32_t idx;
+      ReadEx(jMessage, JsonPointers::idx, &idx);
+      actionListener.OnUpdateProperty(rawMsgData, idx);
+    } break;
+
+    case MsgType::PutItem: {
```
Comment (nic11) [inline skymp5-server/cpp/server_guest_lib/PacketParser.cpp:164]: Why here?

**Review by nic11 — COMMENTED** [review]
Several comments more in addition.

And please remove skymp5-server.lnk and package-locks

**Comment by Pospelove** [comment]
Closing in favor of https://github.com/skyrim-multiplayer/skymp/pull/585

**Pull request closed without merging**